scripts/test-cov.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 set -e
 
-node_modules/.bin/nyc node_modules/mocha/bin/_mocha --opts test/mocha.opts `scripts/_get-test-directories.sh`
-node_modules/.bin/nyc report --reporter=json
+runSequentially=""
+
+if [ "$CI" ]; then
+  runSequentially="--i"
```

Comment:
Not sure why this works, but technically it should be either `--runInBand` or `-i`.